randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -43,32 +55,22 @@ Extra - asset_id: 67
   * Normal Door to Green Brinstar Beetom Room/Door to Green Brinstar Main Shaft
```

Comment:
consdiering the amounts of beetoms, wondering whether Green Brinstar Beetom Room should have some health requirements. doing this without bombs for example and badish movement, youre guaranteed for them to suck out a bit of hp

Reply:
I'll put 98 health here just in case something goes wrong with entrance rando, but I really don't think its an issue.

Reply:
50 health